default/methods/task/task_template/job_list.py

```diff
@@ -18,15 +18,7 @@
 
 @routes.route('/api/v1/job/list',
               methods = ['POST'])
-@General_permissions.grant_permission_for(
-    Roles = ['normal_user'],
```

Comment:
Do we want to make this endpoint public? Maybe we need to use the project permissions instead and modify the URL here?

Reply:
Permissions are checked later by the project wrapper
The general permissions thing is only that the user exists, but it looks for a `user` object only not sdk auth.
This always relied on the project check, but to make it more clear I moved it higher up in execution order.

![image](https://user-images.githubusercontent.com/18080164/180487055-8963b617-f6c3-4f3f-925a-ff40915d2c47.png)
https://github.com/diffgram/diffgram/commit/74189fdb7a9ca0fb0594c900891f94b0a097c27d#diff-62217d126c56b801a3c7da723e459396e8a16950bab04f1502d1e507ebf0d5d5R67

I explained a bit about it here
https://github.com/diffgram/diffgram/commit/74189fdb7a9ca0fb0594c900891f94b0a097c27d

Another option reflecting on this now is we could move this to use project,
and then as we add other things in future that could have a separate endpoint, may be clearer permissions

Reply:
I had kind of wanted to just bump to the /project/project_string_id like we normally do but I also didn't want to conflict with upcoming auth changes.  Should we just move all the way to /project/project_string_id now?

Reply:
Ohh I see interesting! Thanks for pointing to the commit, I think we can leave it for now as it is and later we can discuss the "standard" way for permissions checking here. I think there might be cases where permissions can be checked before the actual endpoint handler, but there might also be cases were we'll need to extract more info to compute some other permissions.